lib/gnarails/cli/application.rb

```diff
@@ -13,50 +14,18 @@ class Application < Thor
         pre-built with the same helpful default configuration you can expect from
         any rails project built by The Gnar Company.
 
-        By default, we pass arguments to `rails new` that:
-        - skip test unit (We'll install rspec later)
-        - use postgres over SQLlite,
-        - use Propshaft,
-        - Bundle CSS with cssbundling (using sass)
-        - bundle JS with esbuild
-
-        You should also be able to pass any other arguments you would expect to
-        be able to when generating a new rails app. Use `rails -h` for more
-        information.
+        Learn more about our Rails Configuration here: https://github.com/TheGnarCo/.gnarrc
       LONGDESC
+
       def new(name)
+        create_file ".railsrc", Net::HTTP.get(URI("https://raw.githubusercontent.com/TheGnarCo/.gnarrc/main/rails/7/.railsrc"))
         Kernel.system command(name, options)
+        remove_file ".railsrc"
```

Comment:
Why do we need to remove this file?

Reply:
it feels weird to leave this behind. If I was going to take another pass at this, I'd probably not even download the file and just use ruby to get the raw contents and append them to the `rails new` task manually.